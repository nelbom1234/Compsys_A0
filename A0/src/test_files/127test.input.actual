test_files/127test.input: UTF-8 Unicode text
