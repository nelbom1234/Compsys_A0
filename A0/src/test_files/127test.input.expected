test_files/127test.input: RDI Acoustic Doppler Current Profiler (ADCP)
